README.md

```diff
@@ -6,10 +6,41 @@ This is testing having a remote registry for [Singularity HPC](https://github.co
 
 📖️ Read the [documentation](https://singularity-hpc.readthedocs.io/en/latest/) 📖️
 ⭐️ Browse the [container module collection](https://singularityhub.github.io/shpc-registry/) ⭐️
- 
+
+## Contribution
+
+This registry showcases Singularity HPC (shpc), and provides the default set of containers for its default registry.
+Contributions are very much welcome, so please do submit a pull-request if you'd like more software to be added !
```

Comment:
```suggestion
Contributions are very much welcome, so please do submit a pull-request if you'd like more software to be added, or open an issue to request a new addition!
```

Reply:
I think that's the last little tweak so they are aware of the issue, then we are good for  merge!